io.github.libredirect.frontends-manager.yml

```diff
@@ -0,0 +1,51 @@
+id: io.github.libredirect.frontends-manager
+runtime: org.gnome.Platform
+runtime-version: '42'
```

Comment:
you should build webkist gtk from source to get the right (unsupported) version. Also you should lobby tauri to fix this. They would rush a fix if it was the proprietary platform. For now they are just building something insecure since it requires to use no-longer supported webengine.

Reply:
This runtime is EOL. Use 45.